style.css

```diff
@@ -0,0 +1,318 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+h1 {
+  font-size: 40px;
+  color: #172b4d;
+  font-weight: 700;
+  max-width: 300px;
+  margin-bottom: 16px;
+  margin-top: 68px;
+  line-height: 52px;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  color: #172b4d;
+  margin-top: 0;
+  position: absolute;
```

Comment:
also remove the default margin as this is preventing your body from covering the full screen 
`margin: 0`

Reply:
done